src/NeuroPlots.jl

```diff
@@ -1,5 +1,1278 @@
 module NeuroPlots
 
-# Write your package code here.
 
+using GLMakie
+import Statistics: mean
+import GMT: triangulate
+
+export plot_topography
+
+"""
+    plot_topography(channelList::Vector{String}, channelValues::Vector{Number}; gridSize = 1000)
+
+Plots a topographical map of the head over the desired points 
+given by `channelList` and their assigned `channelValues`, 
+with default grid size being `gridSize = 1000`.
+
+# Example
+```jldoctest
+plot_topography(PlotTopography.my_channel, rand(59))
+```
+"""
+function plot_topography(channelList::Vector{String}, channelValues::Vector{Number}; gridSize = 1000)
+
+    # Basic checking
+    length(channelList) == length(channelValues) || error(
+        "The lengths of channelList($(length(channelList))) and 
+         channelValues($(length(channelValues))) should be the SAME.",
+    )
+
+    # Retrieve current `channelList` information from the standard channel information
+    channelIndex = indexin(channelList, standardChannelLabel)
+    channelXs, channelYs =
+        standardCoordinateX[channelIndex], standardCoordinateY[channelIndex]
+    maxX, maxY = maximum(channelXs), maximum(channelYs)
+    channelPositions = Point2f.(channelXs, channelYs)
+
+    # Extend data beyond the boundary, see: 
+    # https://www.mathworks.com/matlabcentral/fileexchange/72729-topographic-eeg-meg-plot
+    extendData = append_nearest_values(channelXs, channelYs, channelValues)
```

Comment:
original license here appears to be BSD-3, may need to add more license-related documentation at the method definition